CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
```

Comment:
Please, add an entry into the CHANGELOG.md. Since the change may affect the module build it is better to highlight the change.

Reply:
kk